crates/common/flockfile/src/unix.rs

```diff
@@ -206,4 +218,19 @@ mod tests {
             FlockfileError::FromNix { .. }
         );
     }
+
+    #[test]
+    fn check_pid() {
+        let path = NamedTempFile::new().unwrap().into_temp_path().to_owned();
+        let _ = Flockfile::new_lock(&path).unwrap();
+
+        let mut read_lockfile = OpenOptions::new().read(true).open(&path).unwrap();
```

Comment:
For some reason I fail to spot, this line fails, the lockfile being not found;

Reply:
Yes same for me, though when we will replace variable `_` with `_lockfile` it will work.